### PR TITLE
fix: android label layout in article summary

### PR DESCRIPTION
**Pull request by colincclark**

### CODE REVIEW

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/060ed5817e203f2cbcd50d08e0dff73994ec5ec1/index.html

File: packages/article-summary/__tests__/android/__snapshots__/article-summary-with-style.android.test.js.snap [hunk 1]
```diff
@@ -5,7 +5,7 @@ exports[`1. article summary component with a single paragraph 1`] = `
   <View
     style={
       Object {
-        "marginBottom": 0,
+        "marginBottom": 5,
```
Comment (jimhunty) [inline packages/article-summary/__tests__/android/__snapshots__/article-summary-with-style.android.test.js.snap]: I made this 0 five days ago. This cant be right?

**Comment by jimhunty** [comment]
For reference: https://github.com/newsuk/times-components/pull/1370/files

**Comment by colincclark** [comment]
Seems like this has been done and rolled back several times. Needs further investigating

**Pull request closed without merging**